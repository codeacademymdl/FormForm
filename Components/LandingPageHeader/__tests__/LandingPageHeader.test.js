import React from 'react';

import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../LandingPageHeader.component';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Header heading="hello" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
