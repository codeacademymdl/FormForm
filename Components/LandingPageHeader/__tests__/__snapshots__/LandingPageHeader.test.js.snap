// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#4154af",
      "flexDirection": "row",
      "height": 60,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
        "padding": 10,
      }
    }
  >
    hello
  </Text>
</View>
`;
